Lesson02_Task01.py

```diff
@@ -0,0 +1,58 @@
+poem_text = """I wandered lonely as a cloud
```

Comment:
Linter errors:
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:1:1: D100 Missing docstring in public module
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:6:38: W291 trailing whitespace
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:21:11: C812 missing trailing comma
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:35:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:36:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:37:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:41:11: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:44:7: Q000 Double quotes found but single quotes preferred
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:49:1: N400: Found backslash that is used for line breaking
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:50:5: N400: Found backslash that is used for line breaking
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:51:5: N400: Found backslash that is used for line breaking
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:52:5: N400: Found backslash that is used for line breaking
./T-HOMETASKS/L02/l02_h01_MaximFedoseev.py:57:21: W292 no newline at end of file